devsetup/scripts/common.sh

```diff
@@ -33,18 +33,18 @@ function jinja2_render {
     j2_vars_file=$2
 
     /usr/bin/python3 -c "
-import yaml
 import jinja2
+import os
+import yaml
 
 with open('$j2_vars_file', 'r') as f:
     vars = yaml.safe_load(f.read())
 
-with open('$j2_template_file', 'r') as f:
-    template = f.read()
-
-j2_template = jinja2.Template(template)
+loader = jinja2.FileSystemLoader(os.path.dirname('$j2_template_file'))
+env = jinja2.Environment(autoescape=True, loader=loader)
+env.filters['bool'] = bool
```

Comment:
I would avoid using different logic to the ansible bool filter. It will return True for "yes", False for "no", while `bool("no")` will return True. https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/filter/core.py#L89

Reply:
Also expect filtering through bool will be a no-op. `if <expr>` and `if bool(<expr>)` are equivalent AFAIK

Reply:
This is not a perfect solution, I agree. However, I tested it locally and in CI for multinode jobs, and it works. I will try to improve this in follow ups